src/applications/facility-locator/tests/e2e/search.cypress.spec.js

```diff
@@ -1,18 +1,25 @@
 describe('Facility Search', () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route('GET', '/v0/maintenance_windows', []);
+  });
```

Comment:
I would suggest stubbing out other requests here. I believe there's some mock API that the app is running itself, but if you'd like the stubs to all come from the same place, here would be the place to put them.

Reply:
Is this issue specific to the `/find-locations` page?

Reply:
I'm not sure what "this issue" refers to, but as a whole, the app is running a mock API in localhost, but isn't for a prod build, which is what's being tested in Circle.

Reply:
By "this issue" I mean the need to stub out the call to `/v0/maintenance_windows'`.

Reply:
Yes, I've found that it's generally better to include it, so apps won't have to wait on the request.

I'm thinking of adding a custom command to stub out some common requests like `/v0/maintenance_windows` and `/v0/feature_toggles`.

Reply:
This makes me wonder if the maintenance window request is blocking other parts of the page. Is that necessary?

Reply:
Yes; it needs to check if the external backend services that the app depends on are going to be down. If any of those services are down, the app should show the appropriate downtime messaging.